dlt/common/data_writers/writers.py

```diff
@@ -153,12 +154,12 @@ class ImportFileWriter(DataWriter):
 
     def write_header(self, columns_schema: TTableSchemaColumns) -> None:
         raise NotImplementedError(
-            "ImportFileWriter cannot write any files. You have bug in your code."
+            "`ImportFileWriter` cannot write any files. You have bug in your code."
```

Comment:
I can hear this in Marcin's voice :)
nitpick: should be "a bug"...although its funny and i'd like it